cpu/Makefile.include.cortexm_common

```diff
@@ -121,6 +122,8 @@ include $(RIOTCPU)/Makefile.include.$(TOOLCHAIN)
 
 # use the nano-specs of Newlib when available
 ifeq ($(shell $(LINK) -specs=nano.specs -E - 2>/dev/null >/dev/null </dev/null ; echo $$?),0)
+export CFLAGS += -specs=nano.specs
+export CXXFLAGS += -specs=nano.specs
```

Comment:
I think CFLAGS is already included in CXXFLAGS in Makefile.cflags so I think this is not necessary, but I guess it doesn't hurt either.